setup.py

```diff
@@ -51,12 +51,17 @@
 
 
 def here(*paths):
-    return os.path.abspath(os.path.join(os.path.dirname(__file__), *paths))
+    return os.path.relpath(os.path.join(*paths))
```

Comment:
If we want here to join any path (even if it's already an absolute) and provide a path relative to the cwd then yes. Otherwise we should just remove here entirely? (Removing here entirely probably makes the most sense)